app/src/main/res/values/strings.xml

```diff
@@ -75,6 +75,7 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
     <!-- Settings messages -->
     <string name="msg_contact_us">Contact us</string>
     <string name="msg_language">Language</string>
+    <string name="msg_download_my_data">Download My Data</string>
```

Comment:
Make sure to move this string to _Settings messages_ instead of _Report_ in the other languages. There were also some files that moved `TODO Translate` from the previous string to this one.

Reply:
Sorry! I used the translator helper in Android Studio so didn't look at the individual files. Please have a look again.

Reply:
It's ok, but not all files are fixed yet. Review each one again.

Reply:
@HusseinElFeky I don't know how I missed them. I apologise again. Now all are fixed (I think!)

Reply:
Perfect!